### PR TITLE
Expose client user to the app via an 'X-User' header

### PR DESCRIPTION
**Pull request by kgriffs**

### CODE REVIEW

File: falcon_mutualauth/mutualauth.py [hunk 1]
```diff
@@ -41,9 +41,11 @@ class MutualAuthRequest(Request):
 
     def _inject_headers(self):
         peer_cert = self.channel.transport.getPeerCertificate()
-        common_name = peer_cert.get_subject().CN
+        user = peer_cert.get_subject().CN
 
-        roles = self.roles_map.get(common_name, [])
+        roles = self.roles_map.get(user, [])
```
Comment (ehashman) [inline falcon_mutualauth/mutualauth.py:46]: In what context would we have multiple roles?

**Review by ehashman — APPROVED** [review]
LGTM :+1:

File: falcon_mutualauth/mutualauth.py (hunk 1 above)
Comment (fxfitz) [reply falcon_mutualauth/mutualauth.py:46]: https://github.com/RackSec/overcorps/blob/master/overcorps/config.py#L122

Comment (kgriffs) [reply falcon_mutualauth/mutualauth.py:46]: For example, one user might only have "reader" while another might have both "reader" and "writer".

Comment (fxfitz) [reply falcon_mutualauth/mutualauth.py:46]: Doh! Should have used the Account-Service example! https://github.com/RackSec/account-service/blob/master/src/account_service/config.py#L71-L80

**Review by fxfitz — APPROVED** [review]
LGTM too!

**Comment by fxfitz** [comment]
@kgriffs I think the last thing we need to do with this is release a new version to PyPI.

**Pull request merged**